index.html

```diff
@@ -6,15 +6,130 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <link rel="stylesheet" type="text/css" href="css/grid.css">
+    <link rel="stylesheet" type="text/css" href="css/main.css">
+
+    <link href="https://fonts.googleapis.com/css?family=Open+Sans&display=swap" rel="stylesheet">
+    <script src="https://use.fontawesome.com/1e05dfd930.js"></script>
+    
+
     <!-- Add a link to your custom CSS here -->
 
     <title>Responsive grid project</title>
-</head>
+    </head>
 
-<body>
+<body class="col-12 sm-col-12 ">
 
     <!-- Add your markup here -->
+    <nav class="nav-content col-12 sm-col-12 lg-col-12 no-padding" >
+        <div class=" no-padding col-6 sm-col-6 lg-col-6 left">
+            <div class="left-side col-12 sm-col-12 lg-col-12 no-padding">
```

Comment:
It looks like we have a `col-12` in a `col-6` in a `col-12`...

The grid system is meant to be `.container => .row => .col-number` - it might be worth looking at simplifying this.